caddyhttp/proxy/reverseproxy.go

```diff
@@ -13,6 +13,7 @@ package proxy
 
 import (
 	"crypto/tls"
+	"github.com/mholt/caddy/caddyhttp/httpserver"
```

Comment:
I feel like goimports would move this import line by the other non-std packages... can you double-check that? Run goimports or set up your editor to run it on save. :+1: